docs/transpile/customizing-transpilation.ipynb

```diff
@@ -0,0 +1,414 @@
+{
```

Comment:
<blockquote>Qiskit includes functions to build preset <code><a href="https://github.com/api/qiskit/qiskit.transpiler.PassManager" target="_blank">PassManager</a></code> objects.&nbsp;These preset passmanagers are used by the <code><a href="https://github.com/api/qiskit/qiskit.utils.QuantumInstance#transpile" target="_blank">transpile</a></code> function&nbsp;for each optimization level. There are 4 optimization levels ranging from 0 to 3, where higher&nbsp;optimization levels take more time and computational effort but may yield a&nbsp;more optimal circuit.&nbsp;Optimization level 0 is intended for device characterization experiments and, as such, only&nbsp;maps the input circuit to the constraints of the target backend, without&nbsp;performing any optimizations. Optimization level 3 spends the most effort to optimize the circuit.&nbsp;However, as many of the optimization techniques in the transpiler are heuristic based, spending more&nbsp;computational effort does not always result in an improvement in the quality of the output&nbsp;circuit.</blockquote><p>I think this info is already covered in another page, so you can probably cut it down or remove it entirely. For this intro I think it would be better to indicate why the reader might want to work directly with preset passmanagers in the first place, and give an outline of what will be covered in the whole page</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/391/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Done.